### DISCUSSION TITLE
Welcome, Juan!

### DISCUSSION THREAD

**Discussion by anjutares** [ParanoidUser/yolo#837] [Subscribe]
Hello, I would like to become a collaborator in order to participate in changeme.md ty ;)

**Comment by ParanoidUser** [comment]
Ahoy, @anjutares!

Woohoo! 🎉🎊 Thank you for your interest in collaboration! Your name is now officially [on the list](https://github.com/ParanoidUser/yolo/blob/main/docs/CHANGELOG.md). 🥳

We're always looking for new and innovative ideas to share with the community. Let us know if you're interested in actively contributing to the project. 🤓
